pyglider/website.py

```diff
@@ -11,6 +11,11 @@
 
 import logging
 
+# jpnote addition for argo float calcs 
+import argopandas as argo 
+import pandas as pd
+# end of jp addition 
```

Comment:
These all look useful, but I think we need to move out of pyglider.

Reply:
The argopanda commit is still in this PR...

For website.py, lets copy the version on `upstream/main` into cproofwebsite, and commit it there (maybe under scripts/ or something).  Then make a pr on the website repo for the changes proposed here.  The website can have argopandas as a dependency (not a submodule ;-)) and we can have the map plotting in there etc.  

Probably the same with mapplotting.py. That is far too specific to our sites to be part of pyglider.

Again, we really want pyglider to be of general use, so we should try and move cproof specific routines out if we can.

Reply:
Ah, yes that was an empty argopandas directory that's now been removed.

For website.py and mapplotting.py, can I finish this pull request and then move them over into cproofwebsite? It would be nice to not be trying to do two things at once, when I'm struggling with git for both of them.

Reply:
I don't want to add mapplotting.py to git history of this repo (though its not a big deal).  Just remove it from the PR, and move the file to a temperorary location.  

We could merge the changes to website.py, but you may as well just do the same thing - move the version you want to use to a different location.  Why?  Because we know `website.py` works as-is.  When we move it over, it is _far_ better to start with a version that we know works, than one that hasn't been tested yet, and then make PRs on that version (i.e. the changes you propose here).  The advantage is that we can always back the changes out.

Reply:
Done.